tools/setup/emoji/build_emoji

```diff
@@ -166,8 +168,8 @@ def dump_emojis(cache_path):
 
     emoji_map = json.load(open('emoji_map.json'))
     # Fix data problem with red/blue cars being inaccurate.
-    emoji_map['blue_car'] = emoji_map['red_car']
-    emoji_map['red_car'] = emoji_map['oncoming_automobile']
+    # emoji_map['blue_car'] = emoji_map['red_car']
+    # emoji_map['red_car'] = emoji_map['oncoming_automobile']
```

Comment:
Both red_car and blue_car are gone from the filtered set. Given {their problematic nature, the fact that they behave differently for google emoji than others, the fact that there are plenty of other color mistakes}, seemed better to leave them mapped to what we'll want them mapped to once we upgrade our emoji set? 

Don't have strong feelings here though. I also disagree with the choice of oncoming automobile for `:red_car:`, though that is minor.